### PR TITLE
KG - Change Arm Default Selection

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

File: app/helpers/participant_helper.rb [hunk 1]
```diff
@@ -75,8 +75,14 @@ def statusFormatter participant
   end
 
   def notes_formatter participant
-    content_tag(:button, class: 'btn btn-primary btn-xs participant_notes list notes', 'data-notable-id' => participant.id, 'data-notable-type' => 'Participant') do
-      content_tag(:span, '', class: "glyphicon glyphicon-list-alt")
+    if participant.notes.any?
```
Comment (kyle-glick) [inline app/helpers/participant_helper.rb]: Wat?

File: spec/support/ajax.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ def wait_for_ajax
   end
 
   def finished_all_ajax_requests?
-    page.evaluate_script('jQuery.active').zero?
+    page.evaluate_script('jQuery.active') == 0
```
Comment (wtholt) [inline spec/support/ajax.rb]: Why did you change this?

File: spec/features/participants/participant_list/identity_creates_participant_spec.rb [hunk 3]
```diff
@@ -3,47 +3,83 @@
 feature 'User creates Participant', js: true do
```
Comment (wtholt) [inline spec/features/participants/participant_list/identity_creates_participant_spec.rb]: I'm not a fan of this testing style. Capybara already reads like English. 

Also, is a feature spec absolutely necessary here? SPARC-Fulfillment test suite runs painfully slow as it is.

File: app/controllers/participants_controller.rb [hunk 4]
```diff
@@ -31,8 +31,12 @@ def show
 
   def create
     @participant = Participant.new(participant_params)
+
```
Comment (wtholt) [inline app/controllers/participants_controller.rb]: Any need to add controller specs here?
